ds-cv-inference/mqtt/mqtt.go

```diff
@@ -56,7 +57,7 @@ func NewMqttConnection(connectionString string) Connection {
 	return mc
 }
 
-//define a function for the default message handler
+// define a function for the default message handler
 var commandTopicFunction MQTT.MessageHandler = func(client MQTT.Client, msg MQTT.Message) {
 
 	var edgeXMessage map[string]string
```

Comment:
The payload will now be empty for `get` commands and will fail the json.Unmarshal. I think we can remove this.